src/plugins/discover/public/application/main/discover_main_route.tsx

```diff
@@ -93,13 +93,6 @@ export function DiscoverMainRoute(props: Props) {
           return;
         }
 
-        const defaultDataView = await data.dataViews.getDefaultDataView();
```

Comment:
This functionality duplicates what `dataViews.hasData` is doing. If it doesn't work, it should be fixed.